internal/pkg/vaultworker/tomlParser.go

```diff
@@ -26,24 +26,31 @@ type tomlConfig struct {
 }
 
 type secretservice struct {
-	Scheme               string
-	Server               string
-	Port                 string
-	CAFilePath           string
-	CertPath             string
-	CertFilePath         string
-	KeyFilePath          string
-	VaultInitParm        string
-	VaultSecretShares    int
-	VaultSecretThreshold int
-	TokenFolderPath      string
-	PolicyPath4Admin     string
-	PolicyName4Admin     string
-	TokenName4Admin      string
-	PolicyPath4Kong      string
-	PolicyName4Kong      string
-	TokenName4Kong       string
-	SNIS                 string
+	Scheme                string
+	Server                string
+	Port                  string
+	CAFilePath            string
+	MongoSecretPath 	  string		
+	CoredataSecretPath  	string
+	MetadataSecretPath 		string
```

Comment:
this doesn't look like it is formatted with gofmt

Reply:
fixed.